### PR TITLE
Add container for websocket server

### LINKED ISSUE #342: Add a container for eth_subscribe
Now that we have support for eth_subscribe we want to add a container for it, relay epic: https://github.com/hashgraph/hedera-json-rpc-relay/issues/889

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
**Description**:
Adds a separate container for the Websocket server, which handles `eth_subscribe` and `eth_unsubscribe`.

**Related issue(s)**:

Fixes #342 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -594,6 +595,14 @@ JAVA settings for the Consensus node
 - `RELAY_INPUT_SIZE_LIMIT`: The function input size limit in mb. Defaults to 1.
 - `DEV_MODE`: Allows the asserting of contract call revert messages
 
+### JSON RPC Relay Websocket settings
```
Comment (Nana-EC) [inline README.md:598]: Q: for consistency should we add `RELAY_` for these?

**Review by Nana-EC — COMMENTED** [review]
Looking good.
Needs rebasing as a lot of bumps were done., then possible consideration of `RELAY_` prefix for env variables for consistency

**Comment by Nana-EC** [comment]
@Ivo-Yankov for this is there a way to just have the relay container serve both HTTP and WS so an extra container isn't needed?
In production it's good to have them separate but for local node might be preferred to have them in one

**Comment by Ivo-Yankov** [comment]
> @Ivo-Yankov for this is there a way to just have the relay container serve both HTTP and WS so an extra container isn't needed? In production it's good to have them separate but for local node might be preferred to have them in one

I don't think it's possible with the current images. Maybe if we modify the charts in the relay.

**Pull request merged**